src/main/java/org/folio/circulation/support/ItemRepository.java

```diff
@@ -361,4 +362,56 @@ private ItemRepository(
       .thenComposeAsync(this::fetchMaterialType)
       .thenComposeAsync(this::fetchLoanType);
   }
+
+  public <T extends ItemRelatedRecord> CompletableFuture<Result<MultipleRecords<T>>> fetchItems(
```

Comment:
What does this method do? The name doesn't really tell me which items are fetched.

Reply:
renamed